backend/backend/core/tools.py

```diff
@@ -1,4 +1,5 @@
 import os
+import re
```

Comment:
Importing re module without using it is unnecessary and can cause issues if not properly handled.